api/modules/users/views.py

```diff
@@ -48,7 +50,11 @@ def sign_up(request):
         user.last_name = lastname
         user.is_superuser = False
         user.is_staff = False
+        user.profile.unique_code = UNIQUE_CODE
         user.save()
+        from_email = EMAIL_HOST_USER
+        to_list = [user.username, EMAIL_HOST_USER]
+        send_mail(SUBJECT, MESSAGE, from_email, to_list, fail_silently=True)
```

Comment:
If user credentials were incorrect or expired you are not capturing any error
Add a print statement for logging if there is an error with sending email

Also bring the sending email logic out of except block
That block is already too bulky on it's own